phpBB/styles/prosilver/template/mcp_forum.html

```diff
@@ -96,7 +96,9 @@
 	<!-- ENDIF -->
 
 	<fieldset class="display-options">
+		<!-- EVENT mcp_forum_display_options_prepend -->
 		<label>{L_DISPLAY_TOPICS}{L_COLON} {S_SELECT_SORT_DAYS}</label>
+		<!-- EVENT mcp_forum_display_options_append -->
```

Comment:
Shouldn't the append be at the end of the sorting options instead of right after the dropdown for the timeframe?